.pylintrc

```diff
@@ -10,4 +10,4 @@ max-line-length=120
 # R0801: Similar lines - Imports and method signatures will flag this, such as forwarding action args
 # C0116: Missing method docstring - Adds too much noise
 
```

Comment:
We're missing the list of disables and why in the comments above for these new ones